cli/utils.js

```diff
@@ -120,6 +120,17 @@ const getGraph = (issue, prefix, showFullManifest) => {
         .join('\r\n');
 };
 
+const getIdentifiers = (issue) => {
+    let s = '';
+    for (const ids of Object.values(issue.issueData.identifiers)) {
+        if (ids.length === 0) continue;
```

Comment:
__optional nit:__ it looks like this `if` isn't strictly necessary - the next `for` will basically be a no-op. Or maybe just ``Object.values(issue.issueData.identifiers).flat().map((id) => `- ${id}\r\n`).join('')`` would work?

Reply:
Fixed